opam-packages/0install.json

```diff
@@ -0,0 +1,50 @@
+{
+  "versions": {
+    "2.12.0": {
+      "version": "2.12.0", 
+      "opam": {
+        "url": "https://github.com/0install/0install/archive/v2.12-1.tar.gz"
+      }, 
+      "esy": {
+        "buildsInSource": true, 
+        "build": [
+          "make", 
+          "(opam-installer --prefix=$cur__install || true)"
+        ], 
+        "exportedEnv": {
+          "0install_version": {
+            "scope": "global", 
+            "val": "2.12.0"
+          }, 
+          "0install_installed": {
+            "scope": "global", 
+            "val": "true"
+          }, 
+          "0install_enable": {
+            "scope": "global", 
+            "val": "enable"
+          }
+        }
+      }, 
+      "dependencies": {
+        "@opam/ounit": "*", 
+        "@opam/obus": "*", 
+        "@opam/lwt": "*", 
+        "@opam/react": "*", 
+        "@opam/camlp4": "*", 
+        "@opam/yojson": "*", 
+        "@opam/sha": "*", 
+        "substs": "esy-ocaml-test/substs", 
+        "@opam/extlib": "*", 
+        "@opam/ocurl": "*", 
```

Comment:
Are the versions supposed to be getting dropped?
https://github.com/0install/0install/blob/master/opam#L20